build-on-push/scripts/job-commit-worker.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Copyright 2016 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+PROJECT="$1"
+BRANCH="$2"
+
+. platform-ci/scripts/functions
+
+rm -f "$WORKSPACE/notification-email.txt" "$WORKSPACE/addresses.txt"
```

Comment:
Not sure. On one hand you are right its not needed with the current configuration. On the other, I like being explicit for the possibility the job gets reconfigured to not wipe workspace, for example to increase performance or something.